### ISSUE TITLE
Trying to create a card

### ISSUE THREAD

**Issue by Super-Turtle** [demilich1/metastone#97]
Hello, since the new cards just released for the Whispers expansion, I'v tried to make C'Thun but I' new to this and I'm a little confused. This is what I have so far
{
	"baseAttack": 6,
	"baseHp": 6,
	"battlecry": {
		"spell": {
			"class": "MissilesSpell",
			**"howMany": ????**
		},
		"targetSelection": "ENEMY_CHARACTERS",
		"resolvedLate": false
	},
	"id": "minion_cthun_custom",
	"name": "C'Thun",
	"description": "Battlecry: Deal Damage equal to this minion's Attack randomly split among all enemies.",
	"type": "MINION",
	"heroClass": "ANY",
	"rarity": "LEGENDARY",
	"baseManaCost": 10,
	"collectible": true,
	"attributes": {
		"BATTLECRY": true
	},
	"set": "CLASSIC",
	"fileFormatVersion": 1
}

As you can see I'm not sure what to put into the "howMany" Catagory. If you can help ty!! <3

**Comment by webadict** [comment]
We'll need to add a valueProvider in the spell itself, since I don't think
it allows for it currently. Short answer is I'll do it.

**Comment by Super-Turtle** [comment]
Thanks!!! <3

**Comment by webadict** [comment]
Nah, I actually saw the issue a while ago that fixes this. It's such a ridiculous improvement, I think it was avoided due to how long it takes to implement. But, I'll have the workings of it in about an hour or so.

**Comment by webadict** [comment]
Y'all gonna be so jealous. I've now turned every integer JSON value in Spells except like 2 that don't matter into Values. Values act as both ValueProviders OR integers, depending on what you put into them. Which means it's been increased a ton.

I figure that's what you were trying to do initially, @demilich1, but never got around to it. I've tested it, and it looks like it's working, but it might require a larger testing group.

The only issue might be MissilesSpell, but only if there were some conditional value providers, on the number of times it should fire based on who it targets initially? Basically doesn't matter.
